worker/src/polkadex_db/nonce.rs

```diff
@@ -34,7 +34,7 @@ static NONCE_MIRROR: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
 
 #[derive(Debug)]
 pub struct NonceMirror<D: PermanentStorageHandler> {
-    general_db: GeneralDB<D>,
+    pub general_db: GeneralDB<D>,
```

Comment:
What's the reason for making this pub?

Reply:
Left over from when I was testing with it, removed the pub now.